drivers/SmartThings/philips-hue/src/handlers.lua

```diff
@@ -323,7 +389,9 @@ end
 ---@param device HueDevice
 function handlers.refresh_handler(driver, device, cmd)
   if device:get_field(Fields.DEVICE_TYPE) == "bridge" then
-    do_refresh_all_for_bridge(driver, device --[[@as HueBridgeDevice]])
+    cosock.spawn(function()
+      do_refresh_all_for_bridge(driver, device --[[@as HueBridgeDevice]])
```

Comment:
I mistakenly assumed that Past Doug was smart enough to write this using the bulk REST call and filtering the returned values, but that is not the case; in the name of DRY-ness, this makes 3 API calls per device.

This definitely needs to be fixed soon.